src/openeo_aggregator/utils.py

```diff
@@ -97,10 +97,10 @@ def get(self, key: str) -> Iterator:
             if key in d:
                 yield d[key]
 
-    def union(self, key: str, skip_duplicates=False) -> list:
+    def merge_arrays(self, key: str,skip_duplicates=False) -> list:
```

Comment:
`union` was indeed a poor name, but `merge` is also pretty vague (and `array` is not a common concept in Python). I think `concat` is better choice here